### PR TITLE
Big Checking all new Launchpad Mk3 Extensions / Akai APC

### PR DESCRIPTION
**Pull request by ericahrens**

### CODE REVIEW

File: src/main/java/com/bitwig/extensions/controllers/novation/launchpad_pro/StopClipOverlay.java [hunk 1]
```diff
@@ -3,27 +3,25 @@
...
+      for (int x = 0; x < 8; ++x) {
          final Track track = trackBank.getItemAt(x);
          final Button button = driver.getPadButton(x, 0);
          bindPressed(button, () -> {
             final boolean useAlt = driver.isShiftOn();
-            if (useAlt)
-               track.stopAlt();
-            else
-               track.stop();
+//            if (useAlt)
```
Comment (abique) [inline src/main/java/com/bitwig/extensions/controllers/novation/launchpad_pro/StopClipOverlay.java:16]: Why?

File: src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java [hunk 2]
```diff
@@ -1,227 +0,0 @@
-package com.bitwig.extensions.controllers.akai.mpk_mini_mk3;
```
Comment (abique) [inline src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java:1]: Why did you remove this controller?

**Review by abique — COMMENTED** [review]
I've changed your PR to target the branch api-18 instead of api-17

File: src/main/java/com/bitwig/extensions/controllers/novation/launchpad_pro/StopClipOverlay.java (hunk 1 above)
Comment (ericahrens) [reply src/main/java/com/bitwig/extensions/controllers/novation/launchpad_pro/StopClipOverlay.java:16]: That was a mistake. Accidentally checked that one in. It was only meant for me locally.

File: src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java (hunk 2 above)
Comment (ericahrens) [reply src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java:1]: That was also a mistake. I confused it with APC. It should not be removed.

Comment (abique) [reply src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java:1]: Please double check your changes and let me know when it's good :+1:

Comment (ericahrens) [reply src/main/java/com/bitwig/extensions/controllers/akai/mpk_mini_mk3/MpkMiniMk3ControllerExtension.java:1]: I'll see if I can get it done around noon.

**Comment by ericahrens** [comment]
Reversed the accidental and wrongful changes. Please check again.

**Comment by abique** [comment]
If I'm correct the remote console was removed too, why?

**Comment by ericahrens** [comment]
RemoteConsole is simply not needed any more. It's better to use the host console.

**Comment by abique** [comment]
OK thanks for the changes :+1:
